source/devices/AM62PX/linux/Linux_Performance_Guide.rst

```diff
@@ -193,7 +188,6 @@ Please take note, different run may produce different slightly results.
 This is advised to run this test multiple times in order to get maximum 
 performance numbers.
 
-
```

Comment:
Dhrystone section wasn't updated.
Are the numbers exactly same as SDK 10.0?

Reply:
Yes they seem to be the same